plugins/scaffolder-node/src/tasks/types.ts

```diff
@@ -110,6 +110,7 @@ export type TaskBrokerDispatchOptions = {
  * @public
  */
 export interface TaskContext {
+  taskId?: string;
```

Comment:
Bit of a strange one, but you get this via `.getWorkspaceName()`, see https://github.com/backstage/backstage/blob/372a4cac0d3ba62eccdf1387823b4e417eef9efe/plugins/scaffolder-backend/src/scaffolder/tasks/NunjucksWorkflowRunner.ts#L476

Let's avoid adding this imo, even if it would make a lot of sense 😁

Reply:
Let's keep the PR as is, makes sense to add this as optional for now, and make it required for scaffolder@2?

This should have been in there from the start tbh. 😅